esp-wifi/src/tasks.rs

```diff
@@ -5,6 +5,7 @@ use crate::{
     timer::{get_systimer_count, yield_task},
 };
 
+/// Initializes the `main` and `timer` tasks for the Wi-Fi driver.
 pub fn init_tasks() {
```

Comment:
I think this can be `pub(crate)`?

Reply:
I guess the same as : https://github.com/esp-rs/esp-hal/pull/2354#discussion_r1802610767

Well, I agree, also it doesn't go out far beyond the initial task, so I'll probably make those functions be `pub(crate)` , yes.